cola/models/prefs.py

```diff
@@ -90,6 +91,9 @@ def linebreak():
 def spellcheck():
     return gitcfg.current().get(SPELL_CHECK, False)
 
+def override_tab_with_spaces():
+    return gitcfg.current().get(OVERRIDE_TAB_WITH_SPACES, False)
+
```

Comment:
Sorry to bikeshed, but please follow vim's naming convention for this feature.

We should call it `cola.expandtab`.  The description would be more precise if it said, "Insert spaces instead of tabs" instead of "Override ...".